src/log/tang.jl

```diff
@@ -0,0 +1,389 @@
+# Implementation of
+#  "Table-driven Implementation of the Logarithm Function in IEEE Floating-point Arithmetic"
+#  Tang, Ping-Tak Peter
+#  ACM Trans. Math. Softw. (1990), 16(4):378--400
+#  http://dx.doi.org/10.1145/98267.98294
+# Copyright (c): Simon Byrne
+
+# Does not currently handle floating point flags (inexact, div-by-zero, etc).
+
+
+
+let
+global log_tang
+global log1p_tang
```

Comment:
Out of curiosity, what's the benefit to wrapping everything in `let` and making certain things `global`? I'm not really familiar with that approach.